In [30]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np
%matplotlib inline

# brfss = pd.read_csv('Behavioral_Risk_Factor_Surveillance_System.csv')

# obese = brfss.groupby(['LocationDesc','Data_Value'])['Data_Value'].count()

# print(obese)

ipdr = pd.read_csv('Impaired_Driving_Death_Rate.csv')

# ipdr = ipdr.dropna(axis='columns', inplace=False)
us_ipdr = ipdr.groupby(['State ', 'All Ages, 2014'])['All Ages, 2014'].count()
print(ipdr.keys())

print(us_ipdr)

Index(['State ', 'Location', 'All Ages, 2012', 'All Ages, 2014',
       'Ages 0-20, 2012', 'Ages 0-20, 2014', 'Ages 21-34, 2012',
       'Ages 21-34, 2014', 'Ages 35+, 2012', 'Ages 35+, 2014', 'Male, 2012',
       'Male, 2014', 'Female, 2012', 'Female, 2014'],
      dtype='object')
State           All Ages, 2014
Alabama         5.5               1
Alaska          2.8               1
Arizona         3.0               1
Arkansas        4.6               1
California      2.2               1
Colorado        3.0               1
Connecticut     2.7               1
Delaware        5.2               1
Florida         3.5               1
Georgia         2.7               1
Hawaii          2.2               1
Idaho           3.3               1
Illinois        2.4               1
Indiana         3.2               1
Iowa            3.0               1
Kansas          3.6               1
Kentucky        3.9               1
Louisiana       5.4               1
Maine           3.3               1
Ma

## Problem

3.1 percent of Americans die as a result of impaired driving.  In the subgroup of Georgia, 2.7 percent of individuals die as a result of impaired driving. How can we decrease the percentage of car accidents that cause deaths as a result of impaired driving?

## Potential Solution

I propose that an autonomous taxi service for the inebriated individuals could reduce accidents associated with impaired driving.   

## Test Design

We will take a sample of 1,000 individuals in Georgia who admit to driving after drinking and divide them into 2 equal groups of A and B, respectively.  Group A will get free rides when they are inebriated in an autonomous vehicle for 1 year. We will monitor group B's driving habits in their normal vehicles for one year.  The vehicles of group A and B will be equipped with a device that monitors their driving habits and if any accidents have occurred.  If accidents among group A participants are less that accidents among group B participants, we will determine that autonomous vehicles reduce accidents among the inebriated and will subsequently reduce deaths     